app/lib/main/run_app.dart

```diff
@@ -77,7 +77,10 @@ Future<void> runFlutterApp() async {
   );
 }
 
-Future<AppDependencies> initializeFlutterApp({
+/// Initializes the dependencies of the Flutter app.
+///
+/// Returns the initialized dependencies.
+Future<AppDependencies> initializeDependencies({
```

Comment:
To be honest I think this documentation is unnecessary. It doesn't give any more information than which the name and return type of the method already gives.  

Either delete the Docs or add something which isn't conveyed by the method itself. E.g. why the dependencies are initialized in a seperate method (so that the integration test can use this method) or whatever.

```suggestion
Future<AppDependencies> initializeDependencies({
```

Reply:
Yes, it really does not add any value xD I removed the docs.